docs/manage-metadata-using-gravitino.md

```diff
@@ -730,14 +730,15 @@ The following is the table property that Gravitino supports:
 
 In addition to the basic settings, Gravitino supports the following features:
```

Comment:
Can you add a level 4 title here about `partitioning, bucketing, sort ordering and index` here?